code/datums/jobs.dm

```diff
@@ -746,13 +747,14 @@
 #endif
 	wages = 200
 	slot_back = /obj/item/storage/backpack/withO2
-	slot_belt = /obj/item/device/pda2/engine
+	slot_belt = /obj/item/storage/belt/utility/prepared
 	slot_jump = /obj/item/clothing/under/rank/engineer
 	slot_foot = /obj/item/clothing/shoes/orange
-	slot_lhan = /obj/item/storage/toolbox/mechanical
+	slot_lhan = /obj/item/device/t_scanner
```

Comment:
Why not `slot_poc2`?

Reply:
Doing that forces their pocket money to spawn inside their body

Reply:
Does this happen to other jobs that have `slot_poc2` filled?